app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+  after_create :increment_user_posts_counter
+  def increment_user_posts_counter
+    author.increment!(:posts_counter)
+  end
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
```

Comment:
- [ ] Please kindly fix the MIT link in your README file as it is broken. we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.
![Screenshot (1311)](https://github.com/berylcl/Rails-Blog-app/assets/65298048/ff3559c1-b100-41d1-917d-03437c2c8c3e)